docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1100,9 +1119,13 @@ Default stratifications to all observers should include scenario and input draw.
     - Research team to discuss best approach to resolving issue and will document details. In the meantime, continue running for model versions greater than 6.4 using the specifications from model 6.1 (use birth exposure to calculate the early neonatal LBWSG PAF and use the multiplicative rate to probability calculation) and expect neonatal mortality ratios to be slightly underestimated relative to the GBD verification target.
     - For all runs after 6.4 until research team provides updated documentation
   * - Neonatal mortality due to preterm birth overestimated
-    - Unknown
-    - Research to investigate and come up with plan
-    - N/A
+    - It could be because the docs specify that the ``p_preterm`` parameter should be the prevalence of preterm AT BIRTH, but the artifact data for this key are for every age group except for death. Using a smaller ``p_preterm`` value (which non-birth age groups would be smaller than the birth age group) would result in overestimation of preterm birth cause-specific mortality
+    - Engineering to check how the ``cause.neonatal_preterm_birth.prevalence`` key is being used and update to be specific to birth prevalence if necessary.
```

Comment:
Also, is this using the "Neonatal preterm birth" GBD cause, or is it using the prevalence of preterm categories from the LBWSG risk factor? These should be pretty similar, but the docs specify that we should be using prevalence from the LBWSG frisk factor since we want everything to be consistent with that.